geoconvert/convert.py

```diff
@@ -9,6 +9,7 @@
     DE_POSTCODE_RANGE,
     NUTS_CODES_BY_COUNTRY,
     all_nuts_regex,
+    ambiguous_countries,
```

Comment:
Just a naming suggestion:
```suggestion
    ambiguous_country_names,
```